scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -266,6 +266,7 @@ class ModelDownloadType(Enum):
     FTP = 'ftp'
     GIT = 'git'
     HTTP = 'http'
+    AZURE = 'azure'

```

Comment:
Could we please add notes on implications of using these? 

For example,
`ftp` - to be implemented
`git` - supports only lfs
`http` - to be implemented
`azure` - Anything supported by azcopy

?

Reply:
Also, we could raise errors accordingly

Reply:
added